docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -225,14 +225,14 @@ The following command operators are supported:
     Response headers bytes.
 
   TCP
-    Not implemented ("-").
+    Not implemented (0).
```

Comment:
I see.  we can do that same as %RESPONSE_CODE%;  use ("-")

Reply:
I think the doc here is incorrect. It is still `0` for TCP case.

It is set with `0` instead of `"-"`[here](https://github.com/envoyproxy/envoy/blob/b794aea637bbff4a5b4ba75278eef21912cc4103/source/common/formatter/substitution_formatter.cc#L637) and I verified it by this [test](https://github.com/envoyproxy/envoy/blob/b794aea637bbff4a5b4ba75278eef21912cc4103/test/integration/tcp_proxy_integration_test.cc#L407)

Reply:
OK